File: module-user/src/main/java/com/zhf/user/AttentionActivity.java
package com.zhf.user;

import android.os.Bundle;

import androidx.annotation.Nullable;
import androidx.appcompat.app.AppCompatActivity;

import com.alibaba.android.arouter.facade.annotation.Route;
import com.zhf.common.router.RouterActivityPath;

/**
 * 应用模块:
 * <p>
 * 类描述: 关注页面
 * <p>
 *
 */
@Route(path = RouterActivityPath.User.PAGER_ATTENTION)
public class AttentionActivity extends AppCompatActivity {
    @Override
    protected void onCreate(@Nullable Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.user_activity_attention);
    }
}

File: module-user/src/main/java/com/zhf/user/AttentionActivity.kt
package com.zhf.user

import android.animation.Animator
import android.animation.AnimatorSet
import android.os.Bundle
import androidx.appcompat.app.AppCompatActivity
import com.alibaba.android.arouter.facade.annotation.Route
import com.zhf.common.router.RouterActivityPath

/**
 * created by demoless on 2021/1/29
 * description: 关注页面
 */
@Route(path = RouterActivityPath.User.PAGER_ATTENTION)
class AttentionActivity :AppCompatActivity() {

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.user_activity_attention)
    }
}